client/Index.jsx

```diff
@@ -1,29 +1,66 @@
 import React from 'react';
 import axios from 'axios';
+import styled from 'styled-components';
 import Photogrid from './components/PhotogridComponents/Photogrid.jsx';
+import TopBar from './components/TopBar/TopBar.jsx';
+const CenteringContainer = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  flex-direction: column;
+`;
+
 class Index extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
       photos: null,
+      listingInfo: null,
       listingId: window.location.pathname.slice(1, -1),
     };
   }
 
   componentDidMount() {
-    // Grab the set of photos based on the listing id.
-    axios
-      .get(`http://52.14.166.9:3001/api/photos/${this.state.listingId}`)
-      .then(({ data }) => {
-        this.setState({ photos: data[0] });
+    // Get all information from all services for top bar and photos
+    Promise.all([
+      fetch(`http://52.14.166.9:3001/api/photos/${this.state.listingId}`),
+      fetch(`http://52.14.166.9:4000/api/description/${this.state.listingId}`),
+      fetch(`http://3.12.169.208:2001/api/location/${this.state.listingId}`),
+      fetch(
+        `http://ec2-3-129-14-177.us-east-2.compute.amazonaws.com:8080/api/reviews/${this.state.listingId}`
+      ),
```

Comment:
this needs to be an elastic IP.  hopefully lets set that up tomorrow.

Reply:
Dandy Andy added elastic IPs!